doc/source/io.rst

```diff
@@ -2033,6 +2033,129 @@ using Hadoop or Spark.
   df
   df.to_json(orient='records', lines=True)
 
+
+Table Schema
+''''''''''''
+
+`Table Schema`_ is a spec for describing tabular datasets as a JSON
+object. The JSON includes information on the field names, types, and
+other attributes. You can use the orient ``'table'`` to build
+a JSON string with two fields, ``scehma`` and ``data``.
+
+.. ipython:: python
+
+   df = pd.DataFrame(
+       {'A': [1, 2, 3],
+        'B': ['a', 'b', 'c'],
+        'C': pd.date_range('2016-01-01', freq='d', periods=3),
+       }, index=pd.Index(range(3), name='idx'))
+   df
+   df.to_json(orient='table', date_format="iso")
+
+The ``schema`` field contains the ``'fields'`` keys, which itself contains
+a list of column name to type pairs, including the Index or MultiIndex
+(see below for a list of types).
+The ``schema`` field also contains a ``primary_key`` field if the (Multi)index
+is unique.
+
+The second field, ``data``, contains the serialized data with the ``records``
+orient.
+The index is included, and any datetimes are ISO 8601 formatted, as required
+by the Table Schema spec. You must pass the ``date_format='iso'`` parameter.
+
+The full list of types supported are described in the Table Schema
+spec. This table shows the mapping from pandas types:
+
+==============  =================
+Pandas type     Table Schema type
+==============  =================
+int64           integer
+float64         number
+bool            boolean
+datetime64[ns]  date
```

Comment:
Fixed. I missed datetime at first too (it's under the same header as date) at https://pre-v1.frictionlessdata.io/json-table-schema/. Though I don't see it in the updated http://specs.frictionlessdata.io/table-schema/